sklearn/feature_extraction/text.py

```diff
@@ -1305,7 +1310,7 @@ def idf_(self, value):
 class TfidfVectorizer(CountVectorizer):
     """Convert a collection of raw documents to a matrix of TF-IDF features.
 
-    Equivalent to CountVectorizer followed by TfidfTransformer.
+    Equivalent to ``CountVectorizer`` followed by ``TfidfTransformer``.
```

Comment:
Here you can use single backticks and reference the actual docs of the two classes.

Reply:
Should it be:  
Equivalent to :class:`CountVectorizer` followed by :class:`TfidfTransformer`

or can you point me to an example?

Reply:
yes, if it doesn't work , this should do:
```
:class:`sklearn.feature_extraction.CountVectorizer` 
```

``` git grep \` sklearn/ ``` should also give you more examples.